host-controller/src/main/java/org/jboss/as/domain/controller/operations/SyncModelHandlerBase.java

```diff
@@ -86,7 +85,6 @@ public void execute(OperationContext context, ModelNode operation) throws Operat
         final Resource transformedResource = ReadMasterDomainModelUtil.createResourceFromDomainModelOp(localModel.get(RESULT), localExtensions);
 
         // Create the local describe operations
-        final OrderedChildTypesAttachment orderedChildTypesAttachment = new OrderedChildTypesAttachment();
```

Comment:
I would prefer that this file and SyncModelOperationHandlerWrapper be left alone. It's very complex stuff and I'd rather we see IDE warns about these next time we have to touch them, as that may spark some investigation as to why they were left behind or lead people to understand the history of the code.

Reply:
I will revert the change.